src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReportTest.java

```diff
@@ -1720,4 +1724,767 @@ public void testNoServicesAndNoFiles() throws ParseException {
         // Workloads
         Assert.assertFalse(workloadInventoryReportModel.getSsaEnabled());
     }
+
+
+
+    @Test
+    public void testESXiOS() throws ParseException {
```

Comment:
@m-brophy The same as my previous comment, I think we can refactor all these test methods since the asserts and logic are the same, we can refactor these tests and use parameters.
Methods tha can be refactored: `testESXiOS, testUbuntuOS, testPhotonOS, testFreeBSDOS, testSuseOS, testSolarisOS, testXPOS, testDebianOS`

Reply:
@carlosthe19916 add these details to the issue in Jira as well, thanks

Reply:
@mrizzi @m-brophy I've created the ticket https://issues.redhat.com/browse/MIGENG-428 for refactoring the code.